### PR TITLE
Improve odo create command output

### PR DESCRIPTION
**Pull request by cdrage**
Improves the logging a little bit / output so it's clearer to the user.

Closes: https://github.com/redhat-developer/odo/issues/56

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -120,27 +128,29 @@ A full list of component types that can be deployed is available using: 'odo com
...
 			checkError(err, fmt.Sprintf("failed to push component: %v", componentName))
 
-			fmt.Printf("Component '%s' was created.\n", componentName)
-
 		} else {
 			// we want to use and save absolute path for component
 			dir, err := filepath.Abs("./")
 			checkError(err, "")
+
+			fmt.Printf("Building component %s from directory %s", componentName, dir)
```
Comment (surajnarwade) [inline cmd/create.go]: add `\n` at the end of line

**Comment by surajnarwade** [comment]
rather than one small comment, LGTM

File: cmd/create.go [hunk 2]
```diff
@@ -93,22 +93,30 @@ A full list of component types that can be deployed is available using: 'odo com
...
 			err = component.RebuildGit(client, componentName)
 			checkError(err, "")
+
 		} else if len(componentLocal) != 0 {
+
 			// we want to use and save absolute path for component
 			dir, err := filepath.Abs(componentLocal)
 			checkError(err, "")
+
+			fmt.Printf("Building component %s from local directory %s", componentName, dir)
```
Comment (kadel) [inline cmd/create.go]: missing `\n` 
there should be proabbly `\n\n` to make it same as in `--git` component

File: cmd/create.go [hunk 3]
```diff
@@ -120,27 +128,29 @@ A full list of component types that can be deployed is available using: 'odo com
 				os.Exit(1)
 			}
 
+			fmt.Printf("Building component %s from binary %s", componentName, path)
```
Comment (kadel) [inline cmd/create.go]: missing `\n\n`

File: cmd/create.go (hunk 2 above)
Comment (kadel) [reply cmd/create.go]: Wouldn't it be better without world "component"? Don't know, just something to think about ;-)

File: cmd/create.go [hunk 4]
```diff
@@ -93,22 +93,30 @@ A full list of component types that can be deployed is available using: 'odo com
 			os.Exit(1)
 		}
 
+		fmt.Printf("Creating component %s from build type %s\n", componentName, componentType)
```
Comment (kadel) [inline cmd/create.go]: How about "Creating %s component called %s" ?
So it the resulting string would be something like this:

Creating php component called frontend.
or
Creating nodejs component called nodejs.

**Comment by cdrage** [comment]
@kadel Updated!

File: cmd/create.go [hunk 5]
```diff
@@ -120,27 +128,29 @@ A full list of component types that can be deployed is available using: 'odo com
 				os.Exit(1)
 			}
 
+			fmt.Printf("Building component %s from binary %s\n\n", componentName, path)
```
Comment (kadel) [inline cmd/create.go]: This message is different than the similar one for local component (line 112).
We should align it together with the message in line 144.

Comment (cdrage) [reply cmd/create.go]: Bit confused here... Lines 112 and 144 are the same / similar?

fmt.Printf("Building component %s from directory %s\n", componentName, dir)
fmt.Printf("Building component %s from binary %s\n\n", componentName, path)

**Comment by cdrage** [comment]
I added an extra `\n` but I'm a little confused what you mean by changing the lines? Both 144 and 112 are the same / very similar.

File: cmd/create.go (hunk 5 above)
Comment (kadel) [reply cmd/create.go]: message on line`112` for the local component doesn't have word "component" in it.
But the others (for binary and local) have. I think they should all have the same wording.

**Comment by kadel** [comment]
fmt.Printf("Building component %s from Git repository: %s\n\n", componentName, componentGit)
fmt.Printf("Building %s from local directory %s\n", componentName, dir)

**Comment by kadel** [comment]
sorry for an accidental close, wrong button 😳

**Comment by cdrage** [comment]
@kadel I see! Updated :+1:

**Pull request closed without merging**